File: ProAgil.Domain/Event.cs
using System;
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime Date { get; set; }
        public string Subject { get; set; }
        public int Amount { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<Lot> Lots { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; private set; }
    }
}

File: ProAgil-App/src/app/_models/Event.ts
import { Lot } from "./Lot";
import { SocialNetwork } from "./SocialNetwork";
import { Speaker } from "./Speaker";

export class Event {

  constructor() { }

  id!: number;
  local!: string;
  date!: Date;
  subject!: string;
  amount!: number;
  imageUrl!: string;
  phone!: string;
  email!: string;
  lots!: Lot[];
  socialNetworks!: SocialNetwork[];
  speakerEvents!: Speaker[];

}
